libnavigation-core/src/main/java/com/mapbox/navigation/core/MapboxNavigation.kt

```diff
@@ -1065,6 +1061,7 @@ class MapboxNavigation(
                 USER_AGENT,
                 BuildConfig.NAV_NATIVE_SDK_VERSION,
                 isFallback,
+                navigationOptions.routingTilesOptions.tilesVersion,
```

Comment:
Are we passing the version twice? How's this different than the `tilesVersion` param?

Reply:
cc @mskurydin @korshaknn

Reply:
I think they could be different if fallback is used https://github.com/mapbox/mapbox-navigation-android/blob/f0399b6d1fd04adc25c0434f6ad8eec7a34ad938/libnavigation-core/src/main/java/com/mapbox/navigation/core/MapboxNavigation.kt#L899-L905

Reply:
Reading the `TileEndpointConfiguration` we seem to have `version` and `versionBeforeFallback` :+1:

nit: maybe both should be passed as params for clarity?

Reply:
We may specify one version in the endpoint configuration that corresponds to a chosen off-line region and the original version here to return to (usually, the original version).

Reply:
BTW @korshaknn suggested that above https://github.com/mapbox/mapbox-navigation-android/pull/4478#discussion_r645620565